### ISSUE TITLE
CSS/JS formatting, brackets unexpected behavior

### ISSUE THREAD

**Issue by MaxDesignFR** [panoply/esthetic#32]
- CSS formatting in `{% style %}` renders as such with debatable position of brackets (same behavior for JS brackets opening on a separate line):

![image](https://user-images.githubusercontent.com/46461193/192848838-70602f74-f7eb-45cb-9d5d-c51f1c3471e0.png)

- The "compressCSS" rule also behaves unexpectedly, which is probably the same issue. (Feedback note: I appreciate that the "compressCSS" rule keeps every different css rule on a separate line, it's much more readable than a single one liner) :

![image](https://user-images.githubusercontent.com/46461193/192849250-0535c1d2-d0ef-476e-a1d2-755bc0aecdc6.png)

**Comment by panoply** [comment]
@MaxDesignFR This might be due to `braceAllman` ruleset. Will investigate and try get a patch for tomorrow. Thanks for taking the time to submit these. Huge help.

**Comment by MaxDesignFR** [comment]
> @MaxDesignFR This might be due to `braceAllman` ruleset. Will investigate and try get a patch for tomorrow. Thanks for taking the time to submit these. Huge help.

Ah I see. So it seems you have mentioned all rules on the readme, but for many of them, we can't find it in the typings declaration you redirect to: https://github.com/panoply/prettify/tree/pre-release/types/rules For instance, I don't see `braceAllman` in `style.d.ts` file, but I have noticed it's the case for multiple rules like that. I went through most of them, really neat job! I believe if this is complete it's good enough if someone goes thourgh the readme attentively, but it would be a "plus" to have the rule options/description as a tooltip in vscode setting file.
